src/WFS.js

```diff
@@ -13,6 +13,7 @@ L.WFS = L.FeatureGroup.extend({
     typeName: '',
     typeNSName: '',
     maxFeatures: null,
+    bbox: '',
```

Comment:
There are should be option filter - user should can set any filter what they want. Not only bbox.